install/kubernetes/cilium/values.yaml

```diff
@@ -199,6 +199,9 @@ updateStrategy:
 # nodes if worker nodes share a common L2 network segment.
 autoDirectNodeRoutes: false
 
+# -- Annotate k8s node upon initialization with Cilium's metadata.
+annotateK8sNode: true
```

Comment:
same question in other PR, any reason to enable this by default ?

Reply:
oh got it

> Since this option only existed to set up annotations in Kubernetes Nodes
before the introduction of CiliumNodes, contrary to the upstream commit
this option will be kept to 'true' with the possibility for users to
change it to 'false'.